tests/tests/cluster/test-pizzas.spec.js

```diff
@@ -0,0 +1,229 @@
+    cy.getIframeBody().as('iframe');
+
+    cy.get('@iframe')
+      .contains('Status')
+      .should('be.visible');
+
+    cy.get('@iframe')
+      .contains(/^ready$/i)
+      .should('be.visible');
+
+    cy.get('@iframe')
+      .contains('button:visible', 'Edit')
+      .click();
+
+    cy.getIframeBody()
+      .find('.fd-dialog__content')
+      .as('form');
+
+    cy.get('@form').contains('Name');
+    cy.get('@form').contains('Labels');
+    cy.get('@form').contains('Annotations');
+    cy.get('@form').contains('Description');
+    cy.get('@form')
+      .find('[data-testid="Status"]:visible')
+      .find('input')
+      .type(`{backspace}{backspace}{backspace}{backspace}{backspace}`)
+      .type('Error');
+
+    cy.get('@form').contains('Status');
+    cy.get('@form').contains('Order details');
+    cy.get('@form').contains('Pizzas');
+    cy.get('@form')
+      .find('.fd-form-label--required:visible')
+      .should('have.length', 3);
+
+    cy.get('@form')
+      .contains('button:visible', 'Update')
+      .click();
+
+    cy.get('@iframe')
+      .contains('span', /^READY$/i)
+      .should('not.exist');
+
+    cy.get('@iframe')
+      .contains('span', /^ERROR$/i)
+      .should('be.visible');
+  });
+
+  it('Displays the Pizzas list/detail views from the samples', () => {
+    cy.getIframeBody().as('iframe');
+
+    cy.get('@iframe')
+      .contains('a', 'pizzas/diavola')
+      .click({ force: true });
+
+    cy.get('@iframe')
+      .contains('Hot salami, Pickled jalapeños, Cheese')
+      .should('be.visible');
+
+    cy.get('@iframe')
+      .contains('Diavola is such a spicy pizza')
+      .should('be.visible');
+
+    cy.getLeftNav()
+      .contains(/^Pizzas$/)
+      .click();
+
+    cy.get('@iframe')
+      .find('.fd-table__body')
+      .find('tr')
+      .should('have.length', 2);
+
+    cy.get('@iframe').contains('Margherita is a simple, vegetarian pizza.');
+    cy.get('@iframe').contains('Toppings price');
+  });
+
+  it('Tests the Create Form', () => {
+    cy.getIframeBody()
+      .contains('Create Pizza')
+      .click();
+
+    cy.getIframeBody()
+      .find('.fd-dialog__content')
+      .as('form');
+
+    cy.get('@form')
+      .find('[data-testid="Description"]:visible')
+      .type(PIZZA_DESC);
+
+    cy.get('@form')
+      .find('[data-testid="Sauce"]:visible')
+      .find('input')
+      .type(SAUCE);
+
+    cy.get('@form')
+      .find('[data-testid="Recipe\'s secret"]:visible')
+      .find('input')
+      .type(RECIPE);
+
+    cy.get('@form').contains('Owner References');
+
+    cy.get('@form')
+      .find('[arialabel="Pizza name"]:visible')
+      .type(PIZZA_NAME);
+
+    cy.get('@form')
+      .contains('button', 'Create')
+      .click();
+
+    cy.getIframeBody()
+      .contains('h3', PIZZA_NAME)
+      .should('be.visible');
+  });
+
+  it('Removes the pizza namespace', () => {
+    cy.navigateTo('Back to Cluster Details', 'Namespaces');
+    cy.deleteFromGenericList('pizzas');
+  });
```

Comment:
I wouldn't do the cleanup at all - this is an additional place for fail, and this cluster is purged anyway, after the tests are finished.